src/CPUSolver.cpp

```diff
@@ -763,7 +725,7 @@ void CPUSolver::transferBoundaryFlux(int track_id,
     for (int p=0; p < _num_polar; p++) {
       track_out_flux(p,e) = track_flux(p,e) * bc;
       track_leakage(p,e) = track_flux(p,e) *
-                            _polar_weights(azim_index,p) * (!bc);
+                           _polar_weights(azim_index,p) * (!bc);
```

Comment:
Clearly wasn't intended to be a part of this PR, but is `!bc` a reasonable thing to put in a mathematical expression?  Wouldn't `1-bc` be more clear?